src/dataset-builder/ConceptSearch.ts

```diff
@@ -21,8 +21,9 @@ type ConceptSearchProps = {
   readonly onCommit: (selected: Concept[]) => void;
   readonly onOpenHierarchy: (
     domainOption: SnapshotBuilderDomainOption,
-    selected: Concept[],
-    searchText: string
+    cart: Concept[],
+    searchText: string,
+    openedConcept?: Concept
```

Comment:
Maybe `selectedConcept` instead? `open` to me suggests that the node in the tree itself is open, which isn't true for the selected node.
```suggestion
    selectedConcept?: Concept
```

Reply:
Good suggestion, it was originally `selectedConcept` but @rjohanek brought up some concerns. @rjohanek @pshapiro4broad I'd like to hear everyones thoughts!